src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -120,22 +122,39 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
       refreshSubscriptionTab()
       loadNewChatId()
     }
+    ApplicationManager.getApplication()
+        .messageBus
+        .connect()
+        .subscribe(
+            LafManagerListener.TOPIC,
+            LafManagerListener {
+              ApplicationManager.getApplication().executeOnPooledThread { refreshSubscriptionTab() }
+            })
   }
 
   @RequiresBackgroundThread
   fun refreshSubscriptionTab() {
     CodyAgentService.applyAgentOnBackgroundThread(project) { agent ->
-      if (tabbedPane.tabCount < SUBSCRIPTION_TAB_INDEX + 1) {
-        addNewSubscriptionTab(agent.server)
-      } else {
-        ApplicationManager.getApplication().invokeLater {
-          tabbedPane.remove(SUBSCRIPTION_TAB_INDEX)
+      fetchSubscriptionPanelData(project, agent.server).thenAccept {
+        if (it != null) {
+          ApplicationManager.getApplication().invokeLater { refreshSubscriptionTab(it) }
         }
-        addNewSubscriptionTab(agent.server)
       }
     }
   }
 
+  @RequiresEdt
+  private fun refreshSubscriptionTab(data: SubscriptionTabPanelData) {
+    if (data.isDotcomAccount && data.codyProFeatureFlag) {
+      if (tabbedPane.tabCount < SUBSCRIPTION_TAB_INDEX + 1) {
+        tabbedPane.insertTab("Subscription", null, subscriptionPanel, null, SUBSCRIPTION_TAB_INDEX)
```

Comment:
pls name the `null` params

Reply:
`[NAMED_ARGUMENTS_NOT_ALLOWED] Named arguments are not allowed for non-Kotlin functions`

Reply:
you can `/* my_name= */ null`